src/Data/Number.purs

```diff
@@ -5,12 +5,34 @@ module Data.Number
   , isNaN
   , infinity
   , isFinite
+  , readInt
```

Comment:
I don't think it makes to export `readInt` from this module, as `readInt` doesn't really have much to do with numbers. Additionally, `Data.Int` already provides a `fromString` which does what we want.

Reply:
While it does provide `fromString`, `readInt` can parse a `String` into a specified base. Could we port this function (albeit under a different name) to `purescript-integers`? Or should it be dropped altogether?

Reply:
That's a good point. I think we should just drop it for now; I think we can add integer parsing in a specified base to `purescript-integers` if and when someone asks for it.